tests/test_agg_constraints.py

```diff
@@ -104,6 +106,27 @@ def test_aggregated_node_max_flow(model):
     assert_allclose(A.flow, 20.0)
     assert_allclose(B.flow, 10.0)
 
+
+@pytest.mark.skipif(pytest.config.getoption("--solver") != "glpk", reason="only valid for glpk")
+def test_aggregated_node_max_flow_parameter(model):
+    """Nodes constrained by the max_flow of their AggregatedNode using a Parameter """
+    A = Input(model, "A", max_flow=20.0, cost=1)
+    B = Input(model, "B", max_flow=20.0, cost=2)
+    Z = Output(model, "Z", max_flow=100, cost=-10)
+
+    A.connect(Z)
+    B.connect(Z)
+
+    agg = AggregatedNode(model, "agg", [A, B])
+    agg.max_flow = ConstantParameter(model, 30.0)
```

Comment:
We should test with a parameter that actually changes it's value. I wonder if we should roll these tests up with parameterization?